src/components/Error/Error.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
```

Comment:
Nice to see you've created 404 page also.
I think you've imported react mistakenly.